modules/core-updates/9.0.md

```diff
@@ -388,39 +396,45 @@ We still have some work to do to replace all the usages of the legacy context wi
 
 ## Migrated pages
 
-The migration of pages imply several breaking changes, we won't detail all of them for each page but here is a summary:
+The migration of back office pages to Symfony imply several breaking changes, we won't detail all of them for each page but here is a summary:
 - the legacy URL is no longer reachable, you are automatically redirected to the migrated page
-- the legacy controller is removed along with its associated smarty templates
+- the legacy controller is removed along with its associated Smarty templates
 - the legacy hooks related to controller workflow are no longer called
-- the page no longer uses smarty but twig
+- the page no longer uses Smarty, but Twig
+- the page no longer uses HelperForm and HelperList, but Symfony form components and Grid components
 
 Here is the list of migrated pages in v9:
 
 - International > Locations > States
 - Shop parameters > Order settings > Statuses
 - Orders > Shopping Carts
-- Sell > Catalog > Attributes
-- Sell > Catalog > Feature
-- Sell > Catalog > Products (new version of the page introduce in PrestaShop V8 is now the only available one)
-  - [edition of features use a new optimized data structure in the form](https://github.com/PrestaShop/PrestaShop/pull/34763)
+- Sell > Catalog > Attributes & Features -> Attributes
+- Sell > Catalog >  Attributes & Features -> Feature
+- Sell > Catalog > Products (new version of the page introduce in PrestaShop 8.1 is now the only available one)
```

Comment:
The carrier page should be over soon, we'll have to remember it needs to be added here